cmd/kops/create_cluster.go

```diff
@@ -139,6 +141,9 @@ func NewCmdCreateCluster(f *util.Factory, out io.Writer) *cobra.Command {
 	cmd.Flags().StringVar(&options.OutDir, "out", options.OutDir, "Path to write any local output")
 	cmd.Flags().StringVar(&options.AdminAccess, "admin-access", options.AdminAccess, "Restrict access to admin endpoints (SSH, HTTPS) to this CIDR.  If not set, access will not be restricted by IP.")
 
+	cmd.Flags().StringVar(&options.NodeSecurityGroupIDs, "node-security-groups", options.NodeSecurityGroupIDs, "Add precreated additional security groups to nodes.")
```

Comment:
It might be easier to use a StringSlivceVar here, and make NodeSecurityGroupIDs and MasterSecurityGroupIDs be []string.  I probably should have done that for zones as well..